features/step_definitions/search_bar_steps.rb

```diff
@@ -0,0 +1,49 @@
+  Given('We have the following assignments exist:') do |table|
+    table.hashes.each do |assignment|
+      Assignment.create!(
+        assignment_name: assignment['assignment_name'],
+        repository_name: assignment['repository_name']
+      )
+    end
+  end
+
+  Given('I am on the assignments page') do
```

Comment:
This also looks like a potential duplicate

Reply:
Same as `'I am on the {string} page'`

Reply:
To reuse the existing step definition, it needs to be extended. To address this, I have updated the step definition to also support the assignments page, as it previously only handled the login page.